core/templates/dev/head/pages/base.html

```diff
@@ -249,6 +255,9 @@
     <script src="{{TEMPLATE_DIR_PREFIX}}/components/social_buttons/SocialButtonsDirective.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/components/top_navigation_bar/TopNavigationBarDirective.js"></script>
 
+    <script src="{{TEMPLATE_DIR_PREFIX}}/components/summary_tile/CircularImageDirective.js"></script>
```

Comment:
Hi @seanlip yh I noticed that but this line was added by someone else (it comes from the merge conflict I had) rather than from my changes so I presumed it was put there for a reason. I'd be glad to find a more suitable file for it if I can though. What do you think?